components/_cover-photo.scss

```diff
@@ -61,6 +61,15 @@
 .cover-photo-lede {
   @extend %cover-photo-text;
   font-weight: 600;
+  @include margin(0 0 $base-font-size/2 0);
+}
+
+.cover-photo-subtext {
+  @extend %cover-photo-text;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets